src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1425,6 +1425,18 @@
   <data name="UseAsSpanInsteadOfRangeIndexerOnAnArrayCodeFixTitle" xml:space="preserve">
     <value>Use `{0}` instead of Range-based indexers on an array</value>
   </data>
+  <data name="PlatformCompatabilityCheckTitle" xml:space="preserve">
+    <value>Call of platform dependent API</value>
```

Comment:
How about?
```suggestion
    <value>Validate calls to platform dependent APIs</value>
```

Reply:
I am ok with the change, but not sure if it needs to be plural, the warning would show up for only one method/operation invocation/reference. How about `Validate the call to platform dependent API`

Reply:
How about "Validate Platform Compatibility"?

Reply:
@jeffhandley all need to be capitalized?

Reply:
Oh, probably not.  "Validate platform compatibility" works.